javascript/index.js

```diff
@@ -14,52 +14,123 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  //printing the minutes
+  printMinutes();
+  //printing the seconds
+  printSeconds();
+  //printing the milliseconds
+  printMilliseconds();
 }
 
 function printMinutes() {
-  // ... your code goes here
+  // getting the minutes of the chronometer in 2 digits format
+  let min = chronometer.getMinutes();
+  min = chronometer.twoDigitsNumber(min);
+  // putting into the corresponding DOM element
+  minUni.innerHTML = min[1];
+  minDec.innerHTML = min[0];
 }
 
 function printSeconds() {
-  // ... your code goes here
+  // getting the seconds of the chronometer in 2 digits format
+  let sec = chronometer.getSeconds();
+  sec = chronometer.twoDigitsNumber(sec);
+  // putting into the corresponding DOM element
+  secUni.innerHTML = sec[1];
+  secDec.innerHTML = sec[0];
 }
 
 // ==> BONUS
 function printMilliseconds() {
-  // ... your code goes here
+  // getting the milliseconds of the chronometer in 2 digits format
+  let millisec = chronometer.getMilliseconds();
+  millisec = chronometer.twoDigitsNumber(millisec);
+  //putting into the corresponding DOM element
+  milUni.innerHTML = millisec[1];
+  milDec.innerHTML = millisec[0];
+
 }
 
 function printSplit() {
-  // ... your code goes here
+  // Taking the split time from the chronometer method
+  let time = chronometer.splitClick();
+  // Creating a new li with this value and adding it to the DOM
+  let newLi = document.createElement('li');
+  newLi.innerHTML = time;
+  splits.appendChild(newLi);
 }
 
 function clearSplits() {
-  // ... your code goes here
+  // looping while splits has child node and deleting every child node (=li)
+  while (splits.firstChild) {
+    splits.removeChild(splits.firstChild);
+  }
 }
 
 function setStopBtn() {
-  // ... your code goes here
+  // Changing the classes
+  btnLeft.classList.remove('start');
+  btnLeft.classList.add('stop');
+  // Changing the text
+  btnLeft.innerHTML = "STOP";
+  // Calling the startClick method
+  chronometer.startClick(printTime);
```

Comment:
For more coding clarity, it would make more sense that this startClick function call is in the addEventListener.